api/envoy/data/cluster/v2alpha/outlier_detection_event.proto

```diff
@@ -52,7 +52,12 @@ enum OutlierEjectionType {
   // In case upstream host returns certain number of consecutive gateway errors
   CONSECUTIVE_GATEWAY_FAILURE = 1;
   // Runs over aggregated success rate statistics from every host in cluster
-  SUCCESS_RATE = 2;
+  SUCCESS_RATE_EXTERNAL_ORIGIN = 2;
+  // Consecutive local origin failures: Connection failures, resets, etc
```

Comment:
In this docs, can you link "local origin" to the arch overview docs and a section that clearly defines them? I'm concerned that users are going to find all of these config knobs very confusing (I admit I do).